source/assets/js/playground.ts

```diff
@@ -193,15 +194,53 @@ function setupPlayground(): void {
     const copyURLButton = document.getElementById('playground-copy-url');
     const copiedAlert = document.getElementById('playground-copied-alert');
 
-    let timer: undefined | number;
+    type Timer = undefined | number;
```

Comment:
Please put type definitions at the top level and document them.

Reply:
Resolved in https://github.com/sass/sass-site/pull/1177/commits/661dfef9188f7dccc54ad4b78209526cefadb94d